README.md

```diff
@@ -82,8 +82,7 @@ If package will be used in websockets mode additional settings must be applied.
 * Change in application template async-downloads section to use WS js version.
     ```
     <script src="{% static "js/ws_async_downloads.js" %}" id="async-downloads-script"
-        data-url="{% url ... username=request.user.username %}"
-        data-site-url="{{ SITE_URL }}">
+        data-url="{{ request.build_absolute_uri|slice:":-1" }}{% url ... username=request.user.username %}"
```

Comment:
Can we maybe explain why this is necessary here that we have the absolute URL and not just a relative one?

Reply:
da0d3df93fc50be2f0e4a466db62d0c79a3fdfbf

Reply:
```suggestion
        data-url="{{ FULL_WS_URL }}"
```
I think we should go with something like this in the example because actually the example you have will produce a different URL depending on the page you are on. `build_absolute_uri` will give you the full URL of the current page if you give no arguments. `request.build_absolute_uri("/")` would be what you want, but you can't easily do that in a template.

Reply:
I've just had another thought about this - what do you think about the app having to provide the complete URL including the protocol?
So for example, you might do:
```html
<script ... data-url="wss://app.com/ws/downloads/"></script>
```

Reply:
So for that we need to move logic from js into django backend (example: during view render or settings) and there specify `wss` or `ws`. That is caused by our projects often switch protocols during development.
Which approach do you consider more appropriate here?